backend/src/Repository/StoreOwnerBranchEntityRepository.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\StoreOwnerBranchEntity;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @method StoreOwnerBranchEntity|null find($id, $lockMode = null, $lockVersion = null)
+ * @method StoreOwnerBranchEntity|null findOneBy(array $criteria, array $orderBy = null)
+ * @method StoreOwnerBranchEntity[]    findAll()
+ * @method StoreOwnerBranchEntity[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class StoreOwnerBranchEntityRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, StoreOwnerBranchEntity::class);
+    }
+    
```

Comment:
Empty lines before the ending bracket } of the class must be deleted.
An empty line after the ending bracket } of the class must be added.